packages/web-components/src/components/va-pagination/va-pagination.scss

```diff
@@ -55,3 +56,9 @@ button {
   transition: all 0.3s ease-in-out;
   transition-property: color, background-color;
 }
+
+@container (max-width: 640px) { // Variables won't compile. Matching TSX file for now.
```

Comment:
> // Variables won't compile. Matching TSX file for now.

What does this mean?

Reply:
I was having quite a time passing the USWDS `$tablet` variable into this container query. I tried `var(--tablet)` and `$tablet` and `#${tablet}` but they either wouldn't render the CSS rule at all, or failed during compilation. Happy to change it if there's something I'm missing.

Reply:
That's really interesting 🤔 I wonder if that's a Stencil limitation? Thanks for explaining! 

Do you mind just mentioning in the comment that the value is for the `--tablet` variable/property?